relationshiptypes.md

```diff
@@ -68,3 +68,51 @@ A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspac
 ```
 
 The 2 [item types](itemtypes.md) are embedded
+
+## Search methods
+
+### Relationship types containing an entity type
+**/api/core/relationshiptypes/search/byEntityType?type=<:entity-type-label>**
+
+Parameters:
+* The `type` should be the entity type label from the [entity types endpoint](entitytypes.md). It is mandatory. It can occur on either the left or right hand side
+
+A sample search would be /server/api/core/relationshiptypes/search/byEntityType?id=Publication
+
+It would respond with
+```json
+{
+  "_embedded": {
+    "relationshiptypes": [
+      {
+        "id": 10,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
+        "copyToLeft": false,
+        "copyToRight": false,
+        "leftMinCardinality": 0,
+        "leftMaxCardinality": null,
+        "rightMinCardinality": 0,
+        "rightMaxCardinality": null,
+        "type": "relationshiptype"
+      },
+      {
+        "id": 1,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
```

Comment:
This second response looks the same as the first.  Should this be different? The example here it's clear and probably needs to show different relationshiptypes coming back.

Reply:
fixed